Address-book (#1081)
avinashkranjan/Amazing-Python-Scripts: Address-Book/Readme.md

> 

Reviewer: remove these blank lines

# ADDRESS-BOOK

[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)

This project is to create an Address book using python in which the user can add a new contact, edit and delete existing contact and view all the contact using Tkinter and SQlite.
Tkinter library for rendering graphics and SQlite is used for database.

> # Setup instructions

Reviewer: All subheadings after Address-Book heading should be level 2 (##)

* To install the library tkinter we use the pip install command in the command prompt:

> 	pip install tkinter

Reviewer: tkinter comes preinstalled with python, you can remove this section

# Usage

> * In this python project, the user has to click on a button which functions the user wants to access

Reviewer: Can you rewrite it? 
```suggestion
* In this python project, the user has button functionality to trigger that specific function.
```
